src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -462,6 +482,11 @@ protected override void ProcessRecord()
                 return;
             }
 
+            if (_allStats)
```

Comment:
This should go into the BeginProcessing{} block, since it only needs to be run once.

Reply:
Thanks, Added the same.